src/traffic_server/InkAPITest.cc

```diff
@@ -8572,126 +8572,127 @@ EXCLUSIVE_REGRESSION_TEST(SDK_API_TSHttpConnectServerIntercept)(RegressionTest *
-                                                             "proxy.config.http.uncacheable_requests_bypass_parent",
-                                                             "proxy.config.http.parent_proxy.total_connect_attempts",
-                                                             "proxy.config.http.transaction_active_timeout_in",
-                                                             "proxy.config.srv_enabled",
-                                                             "proxy.config.http.forward_connect_method",
-                                                             "proxy.config.ssl.client.cert.filename",
-                                                             "proxy.config.ssl.client.cert.path",
-                                                             "proxy.config.http.parent_proxy.mark_down_hostdb",
-                                                             "proxy.config.ssl.client.verify.server",
-                                                             "proxy.config.http.cache.enable_default_vary_headers",
-                                                             "proxy.config.http.cache.vary_default_text",
-                                                             "proxy.config.http.cache.vary_default_images",
-                                                             "proxy.config.http.cache.vary_default_other",
-                                                             "proxy.config.http.cache.ignore_accept_mismatch",
-                                                             "proxy.config.http.cache.ignore_accept_language_mismatch",
-                                                             "proxy.config.http.cache.ignore_accept_encoding_mismatch",
-                                                             "proxy.config.http.cache.ignore_accept_charset_mismatch",
-                                                             "proxy.config.http.parent_proxy.fail_threshold",
-                                                             "proxy.config.http.parent_proxy.retry_time",
-                                                             "proxy.config.http.parent_proxy.per_parent_connect_attempts",
-                                                             "proxy.config.http.parent_proxy.connect_attempts_timeout",
-                                                             "proxy.config.http.normalize_ae",
-                                                             "proxy.config.http.insert_forwarded",
-                                                             "proxy.config.http.allow_multi_range",
-                                                             "proxy.config.http.request_buffer_enabled",
-                                                             "proxy.config.http.allow_half_open"};
+std::array<std::string_view, TS_CONFIG_LAST_ENTRY> SDK_Overridable_Configs = {
```

Comment:
Here's a place where `std::array` is actually better - the requirement the array *must* be exactly the size of the enum can be enforced statically. This means if some one adds an override and forgets to update this array, it won't even compile.

Reply:
👍